content/cybersecurity/concepts/red-team/red-team.md

```diff
@@ -0,0 +1,30 @@
+---
+Title: 'Red Team'
+Description: 'Red team is a group of people within an organization who try to play the role of cybercriminals and emulate an adversary's attack.'
+Subjects:
+  - 'Cybersecurity'
+Tags:
+  - 'Cybersecurity'
+  - 'Ethical Hacking'
+CatalogContent:
+  - 'paths/scan-systems-with-nmap'
+  - 'introduction-to-ethical-hacking'
+---
+
+**Red Team** is a group of people within an organization who try to play the role of cybercriminals and emulate an adversary's attack. A Red Team’s objective is to improve an enterprise's cybersecurity posture by demonstrating the impacts of successful attacks.
+
+## Red Team In-Use:
+A Red Team might seek to carry out a penetration test in order to model real-world attack techniques and methods. It can usually gain access through theft of user credentials and social engineering techniques. Red teams are usually composed of highly experienced security professionals or ethical hackers. Red teams can help organizations identify misconfigurations, strengthen network security, and increase awareness among employees as to the risk of human error which may compromise the organization’s security.
```

Comment:
```suggestion
A Red Team might seek to carry out a penetration test to model real-world attack techniques and methods. It can usually gain access through the theft of user credentials and social engineering techniques. Red teams are usually composed of highly experienced security professionals or ethical hackers. Red teams can help organizations identify misconfigurations, strengthen network security, and increase awareness among employees as to the risk of human error which may compromise the organization’s security.
```